### PR TITLE
fix: adjust button and card styling

### PR DESCRIPTION
**Pull request by satya164**

### CODE REVIEW

File: src/components/Card/Card.js [hunk 1]
```diff
@@ -65,7 +65,7 @@ class Card extends Component<DefaultProps, Props, State> {
 
   _handlePressIn = () => {
     Animated.timing(this.state.elevation, {
-      toValue: 6,
+      toValue: 8,
```
Comment (ahmedlhanafy) [inline src/components/Card/Card.js]: @satya164 why did you change this value?

Comment (satya164) [inline src/components/Card/Card.js]: @ahmedlhanafy it should be 8 according to guidelines. It was being clipped earlier somehow, so I had it at 6, but seems it's no longer being clipped.

**Comment by ahmedlhanafy** [comment]
@satya164 I didn't check it on IOS earlier but the borders at the top are sharp
screen shot 2016-11-20 at 4 09 55 pm

**Comment by satya164** [comment]
Hmm... weird, might need to wrap it in a view, gimme 5 mins.

**Comment by ahmedlhanafy** [comment]
And I still think that the margin between buttons doesn't match MD docs @satya164

**Comment by satya164** [comment]
@ahmedlhanafy it's not the margin, the buttons need to be at least 88dp wide according to docs. The screenshot you shared has longer text, so the gap is not visible.

**Comment by ahmedlhanafy** [comment]
@satya164 Take a look at this, I don't know if this should be a different component. Or we should remove this restriction from transparent buttons 
screenshot_20161120-161507

**Comment by ahmedlhanafy** [comment]
@satya164 Also the margin between the image and the title is weird on IOS

**Comment by satya164** [comment]
> Also the margin between the image and the title is weird on IOS

Fixed in the theme PR

About the margin, I don't we can do much here unless you remove the min width. Even if you set margin to 0, it'll still not match the screenshot.

Sometimes google doesn't follow it's own guidelines, so hard to say based on screenshots.

**Comment by ahmedlhanafy** [comment]
> About the margin, I don't we can do much here unless you remove the min width. Even if you set margin to 0, it'll still not match the screenshot.

> Sometimes google doesn't follow it's own guidelines, so hard to say based on screenshots.
I guess we can break it here then because it looks off for me, and we will need this in the dialogs as well so we can't have it looking like this

**Comment by ahmedlhanafy** [comment]
@satya164 I think we will still need to `add minWidth` but a smaller one to prevent something like this
screen shot 2016-11-20 at 4 51 48 pm
It should look like this 
screen shot 2016-11-20 at 4 53 02 pm

**Comment by ahmedlhanafy** [comment]
@satya164 I think we should return the old margin again which is 8 instead of 4

**Comment by satya164** [comment]
what should be the value though?

**Comment by satya164** [comment]
@ahmedlhanafy 4 is a better margin actually, all the issues here are because google hasn't documented this properly https://material.google.com/components/buttons.html#buttons-flat-buttons

**Comment by ahmedlhanafy** [comment]
> what should be the value though?

@satya164 honestly I don't know but you can experiment with different values

**Comment by ahmedlhanafy** [comment]
@satya164 Looks much better now
